### ISSUE TITLE
[NBug] SSH key file could not be found

### ISSUE THREAD

**Issue by VitoMilanese** [gitextensions/gitextensions#10631]
### Environment

- Git Extensions 4.0.1.15887
- Build f2567dea28bbf0e2cbcd2a95d4c7200e47d0d671
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.13
- DPI 96dpi (no scaling)

### Issue description

''

```
GitUI.NBugReports.UserExternalOperationException: SSH key file could not be found
 ---> System.IO.FileNotFoundException: SSH key file could not be found
   --- End of inner exception stack trace ---
   at GitUI.Infrastructure.PuttyHelpers.ThrowIfFileNotFound(String filePath, String errorMessage, String heading)
   at GitUI.Infrastructure.PuttyHelpers.StartPageantIfConfigured(Func`1 sshKeyFileLoader)
   at GitUI.CommandsDialogs.FormPush.StartPageant(String remote)
   at GitUI.CommandsDialogs.FormPush.PushChanges(IWin32Window owner)
   at GitUI.CommandsDialogs.FormPush.PushClick(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.PerformClick()
   at System.Windows.Forms.Form.ProcessDialogKey(Keys keyData)
   at System.Windows.Forms.Control.ProcessDialogKey(Keys keyData)
   at System.Windows.Forms.Control.ProcessDialogKey(Keys keyData)
   at System.Windows.Forms.Control.PreProcessMessage(Message& msg)
   at System.Windows.Forms.Control.PreProcessControlMessageInternal(Control target, Message& msg)
   at System.Windows.Forms.Application.ThreadContext.PreTranslateMessage(MSG& msg)
```

Push not working

### Steps to reproduce

Having a local branch and trying to push it for creating a remote branch.
UPD. Even existing remote branches cannot be pushed anymore.

### Did this work in previous version of GitExtensions?

3.*

**Comment by gerhardol** [comment]
Duplicate of https://github.com/gitextensions/gitextensions/issues/10530
Included in upcoming 4.0.2

**Comment by VitoMilanese** [comment]
Found this [comment](https://github.com/gitextensions/gitextensions/issues/10609#issuecomment-1372787548) with a link for downloading a development build (4.0.2.16028) and it resolves the problem.
https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45842693/artifacts
![image](https://user-images.githubusercontent.com/64318287/211773731-c020a634-99f1-4b62-8ec8-17fbb3554dfc.png)
